### DISCUSSION TITLE
f-secure causing permission errors

### DISCUSSION THREAD

**Discussion by onyx-and-iris** [vlang/v#14484] [Installation Issues]
Hi, I had to add v.exe to f-secure quarantine or I kept getting 'access denied' errors. Just letting anyone else know if they use this antivirus!

**Comment by larpon** [comment]
Thanks for the heads up 👍

**Reply by Wajinn** [reply]
In general, I think it would be good for users of V to report false-positives to their and other anti-virus companies, to make sure they update their databases.  False-positives are a huge problem these days for open-source software and projects.

https://www.techsupportalert.com/how-to-report-malware-or-false-positives-to-multiple-antivirus-vendors/

**Reply by medvednikov** [reply]
A better solution would be to uninstall such malware like f-scure.

**Reply by Wajinn** [reply]
> A better solution would be to uninstall such malware like f-scure.

I understand the sentiment.  Unfortunately, as I have seen the issue before with other programming languages and open-source projects, if a continual effort is not made to stop false-positives then often problems increase.  I tend to check around the forums of other languages and projects, and believe me, it can become a major headache for users.

Once a few of the anti-virus companies have identifiable applications or executables in their databases or security experts start writing lots of blogs focusing on a particular language, other companies and IT tend to follow their lead (particularly if the companies are well-known).  It can happen where the false-positive problem becomes rampant, and then becomes very hard to address at that point, just because of the underserved reputation or attempts by questionable parties to create such a bad reputation.

It is probably for the best that V users be protective of their language and tools, to make sure it doesn't get falsely labeled.
